x/lazyledgerapp/client/cli/payformessage.go

```diff
@@ -0,0 +1,81 @@
+package cli
+
+import (
+	"encoding/hex"
+	"errors"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/lazyledger/lazyledger-app/x/lazyledgerapp/types"
+	"github.com/spf13/cobra"
+)
+
+// CmdCreatePayForMessage returns a cobra command that uses the key ring backend
+// and locally running node to create and broadcast a new WirePayForMessage
+// transaction.
+func CmdCreatePayForMessage() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "payForMessage [hexNamespace] [hexMessage]",
```

Comment:
Other Tx are in JSON encoding or hex, too? (I mean on the cli)

Reply:
That is a good question. I'm not sure. In other tx, each argument is parsed individually, and the only ones handling raw bytes are using either amino or the Bech32 address encoding. For example `lazyledger-appd tx decode [amino-byte-string] [flags]` uses the amino byte string, and all addresses are inputted as a Bech32 string. Should it use amino? Something else to handle bytes?

Reply:
> Should it use amino?

NOOOOOOO! please don't :-D 

But seriously, let's go with hex bytes for now. Let's open an issue about the UX/UI of the CLI. I'm also not sure what would be best here. We can address this much later anyway (before mainnet).